12-offer-encoding.md

```diff
@@ -714,6 +715,7 @@ A writer of an invoice:
     `invreq_chain`.
   - if the invoice is in response to an `invoice_request`:
     - MUST copy all non-signature fields from the invoice request (including unknown fields).
+    - SHOULD add a random delay before sending the invoice.
```

Comment:
I think we should add a rationale on this point as reading it from top (this BOLT) it is not easy to get why should be added.
Maybe pointing it out to BOLT 2 rationale or just a sentence.